### PR TITLE
#84 backend connections

### PR DESCRIPTION
**Pull request by PaulMicu**
connecting the back end packages together, and connecting the model with the ui and display the calculated RUL

### CODE REVIEW

File: src/main/java/com/cbms/source/local/DatabaseConnection.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
     //Make sure to set the user and password to the proper values.
     //Credentials should be set to that which you are using on your local DB server.
     private static final String USER = "root";
-    private static final String PASSWORD = "PaulM1994";
```
Comment (rdsaliba) [inline src/main/java/com/cbms/source/local/DatabaseConnection.java:12]: Remove login information

**Review by rdsaliba — CHANGES_REQUESTED** [review]

File: src/main/java/com/cbms/source/local/DatabaseConnection.java (hunk 1 above)
Comment (PaulMicu) [reply src/main/java/com/cbms/source/local/DatabaseConnection.java:12]: fixed in commit d247eb4

**Review by jeffbuonamici — APPROVED** [review]
No changes to request. Everything looks good.

**Comment by JayChouteau** [comment]
There seems to be no description of what this pull request is for. Maybe give just a quick summary even if the title is self-explanatory.

**Comment by PaulMicu** [comment]
> There seems to be no description of what this pull request is for. Maybe give just a quick summary even if the title is self-explanatory.

I forgot, thank you! description has been updated

**Review by rdsaliba — APPROVED** [review]

**Review by JayChouteau — APPROVED** [review]
LEft a comment for changing the desription, and it was done

**Pull request merged**